### DISCUSSION TITLE
Bale collection for cotton bales

### DISCUSSION THREAD

**Discussion by Slegmoer** [Courseplay/courseplay#7337] [Q&A] [answered]
Hey guys

I am having issue with the bale collection mode and cotton bales. the tractor just stops at the edge of the field and does nothing. I have tried with a few tractors and trailers "autoload and normal" all have the same issue. Does the bale collection not work for cotton bales?

**Comment by Tensuko** [comment, marked as answer]
Does not work yet and is more or less a lower priority as it is not a much used feature of FS19.
But we will see if we can get it working, but don't count for it in the next few weeks...
